libs/domain/order/summary/src/summary.component.ts

```diff
@@ -78,11 +78,6 @@ export class OrderSummaryComponent extends OrderMixin(
       )}
       <div class="title">${i18n('order.payment')}:</div>
       <div>${this.order?.payments[0].paymentProvider}</div>
-      </div>
-      <div class="title">${i18n('order.email')}:</div>
-      <div>${this.order?.billingAddress?.email}</div>
```

Comment:
Our US address schema contain this field, but we don't have this country in supported list. I would make this field at least optional for rendering to keep the data consistent
<img width="785" alt="Screenshot 2023-03-15 at 09 43 30" src="https://user-images.githubusercontent.com/15361565/225228337-58d14c0c-1083-4a3c-a736-70c565e42a77.png">

Reply:
It might be, but we should not display emails on `/order` page till we will have authorization and clear rules for it